test/common/testvm.py

```diff
@@ -853,7 +853,7 @@ def interface(self, number=None):
         self.hostnet += 1
         result = {
             "number": self.offset + number,
-            "mac": '52:54:01:{:02x}:{:02x}:{:02x}'.format(mac & 0xff0000 >> 24, mac & 0xff00 >> 16, mac & 0xff),
+            "mac": '52:54:01:{:02x}:{:02x}:{:02x}'.format(mac & 0xff0000 >> 16, mac & 0xff00 >> 8, mac & 0xff),
```

Comment:
while we're changing this line, wouldn't it be clearer to mask the final 8 bits?
`.format((mac >> 16) & 0xff, (mac >> 8) & 0xff, mac & 0xff)`